### ISSUE TITLE
EMIKAT Permission Problem

### ISSUE THREAD

**Issue by p-a-s-c-a-l** [clarity-h2020/emikat#45]
It seems that we've lost all data (HC-LE, Impact) for studies 33 and 35, at least. E.g. see [here](https://service.emikat.at/EmiKatTst/api/scenarios/3189/feature/view.2974/table/data?rownum=1000&filter=SZ_ID=3189). 

Is there any limit for the number of study data that is stored in EMIKAT? Are 'old' studies cleaned up from time to time? If yes, this is something we have to let the user known about.

**Comment by patrickkaleta** [comment]
> It seems that we've lost all data (HC-LE, Impact) for studies 33 and 35, at least. E.g. see [here](https://service.emikat.at/EmiKatTst/api/scenarios/3189/feature/view.2974/table/data?rownum=1000&filter=SZ_ID=3189).
> 

Data is still there. I can see the [rows](https://service.emikat.at/EmiKatTst/api/scenarios/3189/feature/view.2974/table/data?rownum=1000&filter=SZ_ID=3189) when I use my Emikat account (which has different permissions than the `GAST` account used by the CSIS). **So data is not lost, but it seems that there is a permission issue on the Emikat-side, so @humerh will have to investigate this.**

> Is there any limit for the number of study data that is stored in EMIKAT? Are 'old' studies cleaned up from time to time? If yes, this is something we have to let the user known about.

No, AFAIK there is no limit set, but according to Heinrich Studies inside Emikat take up a lot of space. So, I gave @humerh a list of Studies that he could remove from Emikat about a week ago (list contained Studies that have been removed from the CSIS or were created on my local CSIS instance for development purposes). I believe that he has by now removed those unnecessary Studies, **but Studies 33 and 35 were not among them** and as stated above, that data is still there. It's currently just not accessible by the `GAST` account.
